### ISSUE TITLE
Express deprecated res.sendfile(): use res.sendFile() instead

### ISSUE THREAD

**Issue by kLm8** [camomile-project/camomile-server#48]
Fix: 
in controllers/Medium.js line 153:

change:
```javascript
res.status(200).sendfile(pathToFile);
```
to:
```javascript
res.status(200).sendFile(pathToFile);
```

**Comment by hbredin** [comment]
Good catch. Pull request (in develop branch) accepted :-)

**Comment by hbredin** [comment]
Fixed by 4aaabf47db3c5e1766899ff91cf978b65883c093
